src/Tizen.NUI/src/public/XamlBinding/BindableObject.cs

```diff
@@ -44,15 +44,36 @@ public abstract class BindableObject : INotifyPropertyChanged, IDynamicResourceH
         bool applying;
         object inheritedContext;
 
+        private object bindingContext;
+
         /// <summary>
         /// Gets or sets object that contains the properties that will be targeted by the bound properties that belong to this BindableObject.
         /// </summary>
         /// This will be public opened in tizen_5.0 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
         public object BindingContext
         {
-            get { return inheritedContext ?? GetValue(BindingContextProperty); }
-            set { SetValue(BindingContextProperty, value); }
+            get
```

Comment:
Why don't you use `BindingContextProperty`? `BindingContext` also BindableProperty, so it should be implemented with `BindableProperty` to notify changes of value and to support binding to value ....

Reply:
Fixed it.